autolink/postlink/gradleLinker.js

```diff
@@ -54,6 +57,22 @@ class GradleLinker {
     return contents;
   }
 
+  /**
+   * Check the current minSdkVersion specified and if it's lower than
+   * the required version, set it to the required version otherwise leave as it is.
+   */
+  _setMinSdkVersion(contents) {
+    var minSdkVersion = this._getMinSdkVersion(contents)
+    // If user entered minSdkVersion is lower than the default, set it to default.
+    if (minSdkVersion < DEFAULT_MIN_SDK_VERSION) {
+      debugn(`   Updating minSdkVersion to ${DEFAULT_MIN_SDK_VERSION}`)
+      return contents.replace(/minSdkVersion\s{0,}=\s{0,}\d*/, `minSdkVersion = ${DEFAULT_MIN_SDK_VERSION}`)
```

Comment:
You can use `\s*` to match zero or more occurrences of non-breaking white spaces. Up to you :)

Reply:
Ah I see! Didn't even think about that hahaha. I'm off of my laptop so will probably just going to go with the current to save myself changing it :)

Reply:
No problem 😁